tests/test_simpleButler.py

```diff
@@ -615,6 +615,19 @@ def testJsonDimensionRecords(self):
                     # Also check equality of each of the components as dicts
                     self.assertEqual(r_json.toDict(), r.toDict())
 
+    def test_repr_html_(self):
+        """Test the printing of things for ipython/notebooks doesn't crash."""
+        butler = self.makeButler(writeable=True)
+        butler.import_(filename=os.path.join(TESTDIR, "data", "registry", "hsc-rc2-subset.yaml"))
+
+        for dimension in ("detector", "visit"):
+            records = butler.registry.queryDimensionRecords(dimension, instrument="HSC")
+            for r in list(records)[0:1]:
+                for minimal in (True, False):
+                    ret = r._repr_html_()
+                    self.assertTrue(isinstance(ret, str))
```

Comment:
I'd prefer this test was a bit more specific. It could check something like:
```python
self.assertIn(dimension, ret)
```
couldn't it? Then the `assertFalse` below wouldn't be needed either (and that should have been written as `self.assertNotEqual(ret, "")` if it was to be kept).